test/functional/http/get/voters.js

```diff
@@ -0,0 +1,378 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+var getVotersPromise = require('../../../common/apiHelpers').getVotersPromise;
+
+describe('GET /api/voters', function () {
+
+	var validVotedDelegate = node.eAccount;
+	var validNotVotedDelegate = node.gAccount;
+	var validNotExistingAddress = '11111111111111111111L';
+
+	function expectValidVotedDelegateResponse (res) {
+		node.expect(res).to.have.property('address').that.is.a('string');
+		node.expect(res).to.have.property('balance').that.is.a('string');
+		node.expect(res).to.have.property('voters').that.is.an('array').and.have.a.lengthOf.at.least(1);
+		node.expect(res).to.have.nested.property('voters.0.address').that.is.a('string').and.have.a.lengthOf(21);
+		node.expect(res).to.have.nested.property('voters.0.username');
+		node.expect(res).to.have.nested.property('voters.0.publicKey').that.is.a('string').and.have.a.lengthOf(64);
+		node.expect(res).to.have.nested.property('voters.0.balance').that.is.a('string');
+		node.expect(res).to.have.property('votes').that.is.a('number').equal(res.voters.length);
```

Comment:
How do we now that the number of votes and voters are correct? How do we now if the voters array is correct?

Reply:
Lines above are checking the content of voters array